### DISCUSSION TITLE
Can't delete bucket: "The specified bucket does not exist"

### DISCUSSION THREAD

**Discussion by psxReboot** [minio/minio#18286] [Q&A]
![image](https://github.com/minio/minio/assets/108527711/56a10560-bfe9-4897-90d2-a91da19f1841)
![image](https://github.com/minio/minio/assets/108527711/4b7c61a1-0ec8-42c1-b1fc-fec73b4892d6)
Hello. I want to delete the bucket, but Minio says it doesn't exist.
It's been about two days and that bucket is still here.

**Comment by echo-devnull** [comment]
We are experiencing the same thing. I was able to completely remove the bucket by manually deleting the corresponding folder (same as the bucket name) from all minio nodes / volumes.  It's still somewhere, and somehow not actually removed by minio.

Perhaps a bug?

**Reply by vadmeste** [reply]
Mostly upgrading MinIO version to the latest will fix the issue.

**Reply by echo-devnull** [reply]
Cannot go any later then compiling the development version. In which this is still a problem ;-)
